resources/testautomation.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @namespace
+ */
+var test_automation = {
+    /**
+     * Triggers a user-initiated click
+     *
+     * This matches the behaviour of the {@link
+     * https://www.w3.org/TR/webdriver/#element-click|WebDriver
+     * Element Click command}.
+     * 
+     * @param {Element} element - element to be clicked
+     * @returns {Promise} fulfilled after click occurs, or rejected in
+     *                    the cases the WebDriver command errors
+     */
+    click: function(element) {
+        throw new Error("not implemented");
```

Comment:
> So a test would just include this file and call test_automation.click(element). Right? So if a browser wants to override this to prevent it to go through wptrunner would it be reasonable to do it in testharness-report.js?

Just replace the resources/testautomation.js file (i.e., treat it similarly to resource/testharnessreport.js).

> Also regarding the throw error wouldn't it better to return a rejected promise?

Apparently yes. This is me not knowing modern API design. :)